docker/docker-compose-3/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python
+
+RUN wget https://github.com/jwilder/dockerize/releases/download/v0.6.1/dockerize-linux-amd64-v0.6.1.tar.gz \
+    && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-v0.6.1.tar.gz \
+    && rm dockerize-linux-amd64-v0.6.1.tar.gz
+
+COPY . ./app
+
+WORKDIR ./app
+
+RUN pip install mysql-connector-python
```

Comment:
RUN all of the pip commands in one line, or even better- create a requirements.txt